doc/metadata_routing.rst

```diff
@@ -9,26 +9,38 @@ Metadata Routing
 ================
 
 .. note::
-  The Metadata Routing API is experimental, and is not implemented yet for many
+  The Metadata Routing API is experimental, and is not implemented yet for some
   estimators. Please refer to the :ref:`list of supported and unsupported
   models <metadata_routing_models>` for more information. It may change without
   the usual deprecation cycle. By default this feature is not enabled. You can
-  enable this feature  by setting the ``enable_metadata_routing`` flag to
+  enable it by setting the ``enable_metadata_routing`` flag to
   ``True``::
 
     >>> import sklearn
     >>> sklearn.set_config(enable_metadata_routing=True)
 
-This guide demonstrates how metadata such as ``sample_weight`` can be routed
-and passed along to estimators, scorers, and CV splitters through
-meta-estimators such as :class:`~pipeline.Pipeline` and
-:class:`~model_selection.GridSearchCV`. In order to pass metadata to a method
-such as ``fit`` or ``score``, the object consuming the metadata, must *request*
-it. For estimators and splitters, this is done via ``set_*_request`` methods,
-e.g. ``set_fit_request(...)``, and for scorers this is done via the
-``set_score_request`` method. For grouped splitters such as
-:class:`~model_selection.GroupKFold`, a ``groups`` parameter is requested by
-default. This is best demonstrated by the following examples.
+This guide demonstrates how :term:`metadata` can be routed and passed between objects in
+scikit-learn.
+
+Metadata is data that an estimator, scorer, or CV splitter takes into account if the
+user explicitly passes it as a parameter. For instance, some methods in objects have
+been implemented to use `sample_weight`, `classes`  or `groups` for computation if the
+user deviates from the default `None`. If these objects are used in conjunction with
+other objects, in the past there was no single API for passing the metadata around.
+
+With the Metadata Routing API, we can transfer metadata to estimators, scorers, and CV
+splitters using :term:`meta-estimators` (such as :class:`~pipeline.Pipeline` or
+:class:`~model_selection.GridSearchCV`) or routing functions such as
+:func:`~model_selection.cross_validate`. In order to pass metadata to a method such as
+``fit`` or ``score``, the object consuming the metadata, must *request* it. This is done
+via `set_{method}_request()` methods, where `{method}` is substituted by the name of the
+method that request the metadata. For instance, estimators and splitters, that use the
+metadata in their `fit()` method would use `set_fit_request()`, and scorers would use
+the `set_score_request()`.
```

Comment:
for splitters it would have been `set_split_request`, but we have it the default to be requested for `groups` in our splitters, so rather not mention it here to avoid more confusion.

```suggestion
method that request the metadata. For instance, estimators that use the metadata in
their `fit()` method would provide a `set_fit_request()`, and scorers would use the
`set_score_request()`.
```